### DISCUSSION TITLE
4.1.0rc4

### DISCUSSION THREAD

**Discussion by sadpandajoe** [apache/superset#30820] [General]
Hello Superset Community,

This is a call for the vote to release Apache Superset version 4.1.0.

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/4.1.0rc4/

The Git tag for the release:
https://github.com/apache/superset/tree/4.1.0rc4

The CHANGELOG for the release:
https://github.com/apache/superset/blob/4.1.0rc4/CHANGELOG/4.1.0.md

The instructions for updating to the release:
https://github.com/apache/superset/blob/4.1.0rc4/UPDATING.md

Public keys are available at:
https://www.apache.org/dist/superset/KEYS

<hr /><em>This discussion was created from the release <a href='https://github.com/apache/superset/releases/tag/4.1.0rc4'>4.1.0rc4</a>.</em>

**Comment by hscabral** [comment]
There is a scenario here that seems to be a permission bug.

I have one Database connection, where I have a few schemas. The schemas are identical in structure, but differ in data, of course.

I create Gamma users, and I create Gamma roles for different schemas. **Gamma_A** and **Gamma_B**, let's say. The only permission for each of these roles is `schema access on [DB].[db_conn].[schema_A]`, etc.

In my use case, I need these Gamma users with access to SQL Lab. In it, the single schema assigned to the Gamma_A role is the only one that shows in the schema dropdown - this is correct. However, in the SQL query window, **if I change the schema name in my SQL, I'm able to access the tables in other schemas**.

Example:

`SELECT * FROM schema_a.projects` and `SELECT * from schema_b.projects` work equally. This Gamma user, however, only was assigned schema access on `schema_a`.

**Reply by sadpandajoe** [reply]
@hscabral feel free to create an issue on this and add it to the 4.1 board and we can have discussions in the issue.

**Comment by jacob-roldan** [comment]
Is this feature working in 4.1.0rc4?
https://github.com/apache/superset/pull/29989 fix(screenshots): dashboard screenshots do not capture filter state (@fisjac)

One test I did: https://github.com/apache/superset/pull/29989#issuecomment-2456588070

**Comment by yalcinarslan** [comment]
I am getting "The screenshot could not be downloaded. Please, try again later." message when try to download dashboard as image or pdf

**Reply by gpchandran** [reply]
I had this issue in [4.1.0rc3] but works in [4.1.0rc4]

**Reply by yalcinarslan** [reply]
Actually it worked for me too, but after closing and re-open it I got that message. In 4.1.0rc3 I was getting something looping. I don't understand why this happened.

**Comment by gpchandran** [comment]
Hi @sadpandajoe - any idea when 4.1.0 will be released?

**Reply by sadpandajoe** [reply]
@gpchandran I'll probably close out voting and will work with @michael-s-molina to figure out what we'll need to do to cut and push the official release.

**Reply by gpchandran** [reply]
thanks @sadpandajoe , I see 4.1.0 released but not seeing 4.1.0 docker image here [4.1.0]
https://hub.docker.com/r/apache/superset/tags?name=4.1.0
can you please help here

**Reply by scholarsmate** [reply]
+1, I'm also eager to see the 4.1.0 docker image published.

**Reply by andy1xx8** [reply]
hi yes, i see the ver 4.1.0 have been released but not the docker image

**Reply by sadpandajoe** [reply]
there is this issue here: https://github.com/apache/superset/issues/30919. You can follow along here. Just weird that the rc4 image was pushed fine but there is an issue with the official image.
